META.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "unidecode",
   "abstract": "ASCII transliterations of Unicode text",
-  "version": "0.0.4",
-  "maintainer": "Alexander Kuznetsov <alexkuz@gmail.com>",
+  "version": "0.0.6",
+  "maintainer": "Guillermo Narvaja <guillermo.narvaja@radiocut.fm>",
```

Comment:
Can you make it a list instead? So there'd be both of us.

Reply:
Done. Sorry I changed the maintainer before considering sending the PR, now we are both as maintainers.